sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -219,6 +224,14 @@ def main(unused_argv):
       fn_log_handler.close()
 
 
+def flush_fn_log_handler():
+  """Flushes the FnApiLogRecordHandler if it exists."""
+  _LOGGER.error('The Sdk harness will be terminated now.')
```

Comment:
```suggestion
  _LOGGER.error('The SDK harness will be terminated in 5 seconds.')
  time.sleep(5)
```

I would add "...will be terminated in 5 seconds" and some sleep just in case.